src/App.vue

```diff
@@ -110,6 +110,14 @@ export default {
       await this.reloadUserData();
     },
   },
+  metaInfo() {
+    return {
+      // default title
+      title: 'Superhero.com',
```

Comment:
This is not working this way

Reply:
It default fallback for the case when route don't have any meta title